VocaDbWeb/Views/User/Login.cshtml

```diff
@@ -8,7 +7,8 @@
 
 @Helpers.ValidationSymmaryDiv(Res.UnableToLogin)
 
-<form action="@VocaUriBuilder.Absolute(Url.Action("Login", "User"))" method="POST">

```

Comment:
Is there any reason why you use form tag and `VocaUriBuilder.Absolute` instead of `Html.BeginForm` here?

Reply:
Can't be 100% sure, but I think it was because originally VocaDB login form was using https, while the rest of the site was http by default, so the URL would have to be absolute. Now it could be either.